.github/workflows/blackbox-main.yml

```diff
@@ -42,6 +42,18 @@ jobs:
       - name: Prepare
         uses: ./.github/actions/prepare
 
+      - name: Prune Dependencies
+        shell: bash
+        run: find . -type d -name node_modules -prune -exec rm -rf {} \;
+
+      - name: Install Production Dependencies
+        shell: bash
+        run: pnpm install --prod
```

Comment:
Couldn't we use something like `pnpm prune --prod` instead of these two steps? See https://pnpm.io/cli/prune

Reply:
I believe we can't, being a monorepo. 🤔 

<img width="730" alt="image" src="https://user-images.githubusercontent.com/26413686/199500890-2b801ad4-3f50-4967-a783-3353dd20bf80.png">